### ISSUE TITLE
Changing session path to not default.

### ISSUE THREAD

**Issue by kol9n93** [bergie/passport-saml#178]
Hi. I have the following problem: I've changed express session cookie path from default '/' to '/test'. My /login/callback function does redirect to index page. Before redirect I have a user authenticated via passport in request session, but after redirect request session is undefined. 
When session is undefined, app requests to SSO agent, SSO agent calls /login/callback which redirects to index page. Index page doesn't have a session and so on in a loop.
May be you know how to fix it?
I need to run 2 applications under a SSO agent. But they have mixed SSO session when using default cookie path. They should use pathes /app1 and /app2.

**Comment by pdspicer** [comment]
Non-default cookie path will mean that unless your login callback is at /test/login/callback, then the cookie will not be sent with the request, hence why you are probably seeing an undefined session.
